docs/docs/customization/css.md

```diff
@@ -0,0 +1,118 @@
+Pathfora is built to be entirely customizable for developers. With the javascript config alone you can [change the colors](/customization/themes.md) of any element of the module. But, to make your module fully fit the look and feel of your site you can add custom CSS.
+
+## Key Class Names
+
+The outer most `<div>` of all Pathfora modules have the `pf-widget` class. For most modules this will be the the containing div surrounding the content, but for [modal](/layouts/modal.md) and [gate](/types/gate.md) modules this div will contain the full-screen overlay behind the module. `pf-widget` has a number of useful subclasses to help select modules by their settings from the javascript config.
+
+**Subclasses of `pf-widget`:**
+
+| Class Name | Use |
+|---|---|
+| `pf-[type]` | select modules by type |
+| `pf-widget-[layout]` | select modules by layout |
+| `pf-position-[position]` | select modules by position |  
+| `pf-widget-variant-[variant]` | select modules by variant |
+| `opened` | added once the module becomes visiable to the user, can be used for changing the transition animation |
+
+
+Within the `pf-widget` div most key elements are assigned class names. Form components are nested in a `<form>` element and should be selected and styled by element type.
+
+**Key Classes:**
+
+| Class Name | Element |
+|---|---|
+| `pf-widget-content` | second container div inside `pf-widget` (for modal and gate layouts this will the the box surrounding the actual module content) |
+| `pf-widget-headline` | `<h2>` headline text |
+| `pf-widget-message` | `<p>` containing widget message text |  
+| `pf-widget-close` | "x" icon to close the modal |
+| `pf-widget-btn` | general class for all buttons |
+| `pf-widget-ok` | "Confirm" button |
+| `pf-widget-cancel` | "Cancel" button |
+| `pf-widget-img` | image element for modules of [variant 2](/layouts/modal.md#variant)
+| `pf-social-login` | div containing social login buttons for modules with [showSocialLogin](/types/form.md#showsociallogin) enabled | 
+
```

Comment:
It would be great to have an html code snippet here for reference. I think that's a way better way to wrap your head around what css classes apply to which elements.